src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -85,6 +85,23 @@
         results (jdbc/with-db-transaction [] (jdbc/query-to-vec query))]
     (map :sequence_name results)))
 
+(defn sql-current-connection-function-names
+  "Returns the names of all of the functions in the public schema of
+  the current connection's database.  This is most useful for
+  debugging / testing purposes to allow introspection on the
+  database.  (Some of our unit tests rely on this.)."
```

Comment:
I suppose it could be. But I put it here because it's right next to `sql-current-connection-table-names` and `sql-current-connection-sequence-names`.